extensions/terminal-suggest/src/test/completions/code.test.ts

```diff
@@ -39,8 +38,9 @@ export function createCodeTestSpecs(executable: string): ITestSpec[] {
 		{ input: `${executable} --goto |`, expectedResourceRequests: { type: 'files', cwd: testPaths.cwd } },
 		{ input: `${executable} --user-data-dir |`, expectedResourceRequests: { type: 'folders', cwd: testPaths.cwd } },
 		{ input: `${executable} --profile |` },
-		{ input: `${executable} --install-extension |` },
-		{ input: `${executable} --uninstall-extension |` },
+		{ input: `${executable} --install-extension |`, expectedCompletions: ['code'] },
```

Comment:
see this line for why this is the expectation 

https://github.com/microsoft/vscode/blob/e0f606a6670bf0a8e5f5183da5d7aa545471c78c/extensions/terminal-suggest/src/test/terminalSuggestMain.test.ts#L131